content/intro-to-storybook/angular/en/data.md

```diff
@@ -293,42 +290,12 @@ Empty.args = {
   />
 </video>
 
-Similarly, we need to use `PureTaskListComponent` in our Jest test:
-
-```diff:title= src/app/components/task-list.component.spec.ts
-import { render } from '@testing-library/angular';
-
-- import { TaskListComponent } from './task-list.component.ts';
-+ import { PureTaskListComponent } from './pure-task-list.component';
+<div class="aside">
+💡 With this change, all of our tests will require an update. Update the imports and re-run the test command with the <code>-u</code> flag to update them. Also, don't forget to commit your changes with git!
```

Comment:
Are the tests in this tutorial using snapshots? I didn't catch that. If they aren't, then the `-u` flag isn't needed here.

Reply:
@kylegach this is a safeguard as in the past readers ran the testing script and complained about it. So I left the aside.